source/elements/oneDAL/source/glossary.rst

```diff
@@ -0,0 +1,159 @@
+.. _glossary:
+
+=========
+Glossary
+=========
+
+Machine learning terms
+======================
+
+.. glossary::
+    :sorted:
+
+    Categorical feature
+        A :term:`feature` with a discrete set of possible values without ordering.
+
+        Synonyms: nominal feature.
+
+    Classification
+        A :term:`supervised machine-learning problem<Supervised learning>` of assigning
+        discrete :term:`labels<Label>` (:term:`categorical<Categorical feature>` or :term:`ordinal<Ordinal feature>`)
+        to :term:`feature vectors<Feature vector>`.
+
+    Continuous feature
```

Comment:
Its debatable how to name such features. Some sources name it "numerical", some "continuous" or "quantitative". For example, Peter Flach (p.304):
> <...> we can distinguish three main kinds of feature: those with a meaningful numerical scale, those without scale but with an ordering, and those without either. We will call features of the first type quantitative <...> (another term in common use is 'continuous').

Reply:
@rlnx @andreynv what is your opinion on how we should primarly name it? Three of my sources name it differently - Peter Flach calls it "quantitative", Jure Leskovec et all call it "numerical", Google ML glossary - "continuous". I chose the last one because it has been already used in DAAL.

Reply:
responded somewhere else. If we see different terms, we have freedom to stick to one of them